src/components/SearchContainer/SearchContainer.css

```diff
@@ -0,0 +1,20 @@
+.bg-search {
+  background-image: url('../../assets/night.jpg');
+  background-repeat: no-repeat;
+  height: 500px;
+  background-position: center;
+  background-color: #cccccc;
+  background-size: cover;
+  position: relative;
+}
+
+.search-title {
+  position: absolute;
+  top: 50%;
+  left: 40%;
+  transform: translate(-50%, -50%);
+  color: white;
+  font-weight: bold;
```

Comment:
Nit: Through the css files, we use font weights of 100,300,400 and bold. Not all may be available for the specific font used.